docs/components/toast/usage.md

```diff
@@ -27,33 +27,33 @@ We do not recommend placing toasts in any other part of the viewport.
 :::image-block
 
 <div class="mx-16 flex flex-cols">
```

Comment:
None of this html inside of the :::image-blocks should be needed?  Could we maybe use one :::image-block pr image , it would naturally align itself inside of that. and Move any headings  to be normal ## Headings or whatever level is appropriate outside of that block.

Reply:
(imo the Figma sketches are not following the set pattern for this, and should not always be taken literally )